### ISSUE TITLE
with Python3.4 getting Illegal instruction error

### ISSUE THREAD

**Issue by gohilbhagirath90** [Kitt-AI/snowboy#253]
Hi,

I install swig3.0.12.

I download snowboy.git
go to snowboy/swig/Python
make 
sucessfully build _snowboydetect.so and snowboydetect.py

goto snowboy/examples/Python3

Run Example : 
python3 demo.py ./resources/snowboy.umdl

getting error : Illegal instruction

what is issue ?

**Comment by chenguoguo** [comment]
You should go to snowboy/swig/Python3 instead of snowboy/swig/Python

**Comment by gohilbhagirath90** [comment]
I am using snowboy/swig/Python3 

and compile successfully .so file  but i am getting error : Illegal instruction when i run python3 demo.py ./resources/snowboy.umdl with this build library.

my board architecture is armv7 ([ IFC6410 ](https://www.inforcecomputing.com/products/single-board-computers-sbc/qualcomm-snapdragon-600-inforce-6410plus-sbc))

i run on linaro linux release.

**Comment by chenguoguo** [comment]
When you `make` under `snowboy/swig/Python3`, is it using lib/rpi/libsnowboy-detect.a or something else?

**Comment by gohilbhagirath90** [comment]
Yes, it using lib/rpi/libsnowboy-detect.a that is problem. My ifc6410 is armv7 arch which run linaro Debian Ubuntu release.

Can you please provide libsnowboy-detect.a that work on armv7 Debian.

**Comment by chenguoguo** [comment]
Sorry, we don't provide that library yet...

**Comment by gohilbhagirath90** [comment]
Why any reason.

If I want to snowboy without it any idea.

**Comment by chenguoguo** [comment]
You can try to run raspbian on your armv7 arch.
